content/en/monitors/service_level_objectives/monitor.md

```diff
@@ -6,57 +6,73 @@ further_reading:
 
-While the SLO remains above the target percentage, the SLO's status will be displayed in green font. When the target percentage is violated, the SLO's status will be displayed in red font. You can also optionally include a warning percentage that is greater than the target percentage to indicate when you are approaching an SLO breach. When the warning percentage is violated (but the target percentage is not violated), the SLO status will be displayed in yellow font.
+If you need finer granularity than the once a minute monitor evaluation, consider using [metric-based SLOs][3] instead.
 
-**Note:** The number of decimal places you can specify for your monitor-based SLOs differs depending on the time windows you choose. Up to two decimal places are allowed for 7-day and 30-day targets, and up to three decimal places are allowed for 90-day targets. The precision shown in the details UI of the SLO will be up to `num_target_decimal_places`, meaning two decimal places for 7-day and 30-day targets and three decimal places for 90-day targets. These limits are in place because monitor-based SLO error budget is specified in terms of time. A 99.999% target for a 7-day and 30-day targets results in error budgets of 6 seconds and 26 seconds, respectively. Monitors evaluate every minute, thus the granularity of a monitor-based SLO is also 1 minute. This means one alert would fully consume and overspend the 6 second or 26 second error budgets in the previous examples (for use cases requiring more granularity, consider using [metric-based SLOs][6] instead). Furthermore, error budgets this small are likely unrealistic goals for you to try to achieve in practice.
+### Add name and tags
 
-### Identify this indicator
+Choose a name and extended description for your SLO. Select any tags you would like to associate with your SLO.
 
-Here you can add contextual information about the purpose of the SLO, including any related information or resources in the description and tags you would like to associate with the SLO.
+Select **Save & Exit** to save your new SLO.
 
-## Overall status calculation
+## Status calculation
 
-{{< img src="monitors/service_level_objectives/overall_uptime_calculation.png" alt="overall uptime calculation"  >}}
+{{< img src="monitors/service_level_objectives/aggregate_slo.jpg" alt="SLO detail showing 99 percent green with 8 groups aggregated"  >}}
 
-The overall status can be considered as a percentage of the time where **all** monitors or **all** the calculated groups in a single multi-alert monitor are in the `OK` state. It is not the average of the aggregated monitors or the aggregated groups, respectively.
+Datadog calculates the overall status as the percentage of the time where **all** monitors or **all** the calculated groups in a single multi-alert monitor are in the `OK` state. Datadog does not calculate the average of the aggregated monitors or the aggregated groups.
```

Comment:
```suggestion
Datadog calculates the overall status as the percentage of the time where **all** monitors or **all** the calculated groups in a single multi-alert monitor are NOT in the `ALERT` state. Datadog does not calculate the average of the aggregated monitors or the aggregated groups. Monitor-based SLOs will also treat the `WARN` state as `OK`. This is because an SLO's definition requires a binary distinction between good and bad behavior, thus `WARN` is treated as good behavior since it is not severe enough to be considered bad behavior.
```